packages/test/test-utils/src/TestSummaryUtils.ts

```diff
@@ -136,9 +140,18 @@ export async function createSummarizer(
 /**
  * Summarizes on demand and returns the summary tree, the version number and the reference sequence number of the
  * submitted summary.
+ *
+ * @param summarizer - The ISummarizer to use to summarize on demand
+ * @param inputs - Either the reason string or the full IOnDemandSummarizeOptions.
+ * Defaults to the reason "end-to-end test".
  */
-export async function summarizeNow(summarizer: ISummarizer, reason: string = "end-to-end test") {
-	const result = summarizer.summarizeOnDemand({ reason });
+export async function summarizeNow(
+	summarizer: ISummarizer,
+	inputs: string | IOnDemandSummarizeOptions = "end-to-end test",
+) {
+	const options: IOnDemandSummarizeOptions =
+		typeof inputs === "string" ? { reason: inputs } : inputs;
+	const result = summarizer.summarizeOnDemand(options);
```

Comment:
@agarwal-navin is this ok long-term or should I do a follow-up that replaces this with just an `options` param?